src/osdc/Objecter.cc

```diff
@@ -189,82 +189,82 @@ void Objecter::init()
 
     pcb.add_u64(l_osdc_op_active, "op_active",
         "Operations active", "actv");
-    pcb.add_u64(l_osdc_op_laggy, "op_laggy");
-    pcb.add_u64_counter(l_osdc_op_send, "op_send");
-    pcb.add_u64_counter(l_osdc_op_send_bytes, "op_send_bytes");
-    pcb.add_u64_counter(l_osdc_op_resend, "op_resend");
-    pcb.add_u64_counter(l_osdc_op_ack, "op_ack");
-    pcb.add_u64_counter(l_osdc_op_commit, "op_commit");
-
-    pcb.add_u64_counter(l_osdc_op, "op");
+    pcb.add_u64(l_osdc_op_laggy, "op_laggy", "Laggy operations");
+    pcb.add_u64_counter(l_osdc_op_send, "op_send", "Sent operations");
+    pcb.add_u64_counter(l_osdc_op_send_bytes, "op_send_bytes", "Size of sent operations");
+    pcb.add_u64_counter(l_osdc_op_resend, "op_resend", "Resent operations");
+    pcb.add_u64_counter(l_osdc_op_ack, "op_ack", "Acknowledgement callbacks");
+    pcb.add_u64_counter(l_osdc_op_commit, "op_commit", "Operation commits");
```

Comment:
Commit callbacks, to be consistent with Acknowledgement callbacks

(it's the number of times we call back to the caller on commit or commit_sync contexts, rather than the actual number of operations committed)